modules/addbook.js

```diff
@@ -0,0 +1,82 @@
+const collection = document.querySelector('.collection');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const addBtn = document.querySelector('#add-btn');
+let id = 1 || JSON.parse(localStorage.getItem('maxID'));
+
+class BookObject {
```

Comment:
- [ ] You should also implement the` remove book` function as this is a project requirement. Let's fix that

Reply:
But the remove book functionality is there, it works.

Reply:
https://github.com/iBel251/awesome-books-modules/blob/f165efe45702bd3d7d40a1de81a4fd18feb0669e/modules/addbook.js#L37

Reply:
Okay my bad. I missed that, just request for another CR it will be approved!